packages/cli/src/template/scaffold-handlers.ts.ejs

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: Apache-2.0
+
+// Auto-generated
+<% props.abis.forEach(function (abi) { %>
+import {<% abi.handlers.forEach(function (handler) { %><%= handler.argType %>,<% })%>} from "../types/abi-interfaces/<%= abi.name %>";
+<% })%>
+import assert from "assert";
```

Comment:
Assert is unused here, unless you want to assert the args exists within the handler